io.github.aginies.watermark.yml

```diff
@@ -0,0 +1,25 @@
+app-id: io.github.aginies.watermark
+runtime: 'org.freedesktop.Platform'
+runtime-version: '24.08'
+sdk: 'org.freedesktop.Sdk'
+command: watermark_app
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --device=dri
+  - --filesystem=host
+  - --talk-name=org.freedesktop.portal.Notification
```

Comment:
you don't use the portal by adding permissions, you have to modify your code to use it...

eg. libnotify already supports it so if you use it through some binding then it should work automatically

Reply:
this should been addressed.